README.md

```diff
@@ -136,6 +136,37 @@ $ vendor/bin/phpunit
 
 The tests will execute and you'll be presented with a summary.
 
+### Running tests using `wp-env`
+
+To run PHPUnit from within the included `wp-env` environment, you will need to load the development version of the WooCommerce plugin in your environment.
+
+First, clone the WooCommerce repo, replacing `${WC_VERSION}` with the latest version of the plugin (e.g. 8.6.1):
+
+```bash
+$ git clone --depth=1 --branch="${WC_VERSION}" https://github.com/woocommerce/woocommerce.git ../woocommerce/`
```

Comment:
All the instructions assume we are going to place `woocommerce` in the parent directory of `google-listings-and-ads`. Commonly that would be in `wp-content/plugins/`. Considering that we are also placing a copy of the monorepo in that folder and not just the `woocommerce` plugin, I think this construction could generally lead to a bit of confusion.

On my install if I'd have a development version of WooCommerce installed I'd have a symlink in my `wp-content/plugins` directory installed. I placed the full directory in `.wp-env.override.json` so it would point to the right development copy of WooCommerce, but we might need to clarify that in these instructions.

Reply:
Right, this is not as intuitive as it could be. Essentially, I'm trying to avoid adding these files to the same folder that the GLA plugin repo lives, so folks don't have to worry about another `.gitignore`, which is why we check out the mono-repo in the parent directory of the gla plugin. We could alternatively check it out in a special folder in the root directory where GLA is checked out and `.gitignore` that folder instead.

The `.wp-env.override.json` file will only load the development version of the `woocoomerce` plugin from the monorepo in the plugins folder of the environment and not any of the other plugins from the mono-repo. Really, the only reason this is needed is because the unit tests use some of the test classes from the woocommerce development repo as a dependency. If those helpers could be loaded in another way during the PHPUnit bootstrap process, then this could be a bit simpler.

Reply:
Thanks, I'm fine with leaving it as is (I prefer it outside the plugin folder). Just a note to clarify that any location can be used (since a lot of people would already have a copy or I could point it to the copy installed by `bin/install-wp-tests.sh`) and where it needs an updated path should be helpful enough.

Reply:
> If those helpers could be loaded in another way during the PHPUnit bootstrap process, then this could be a bit simpler.

The bootstrap process doesn't really have a lot of heavy requirements, it just loads 4 files directly with the full path: https://github.com/woocommerce/google-listings-and-ads/blob/2.6.0/tests/bootstrap.php#L63-L66

It also checks for the presence of bootstrap.php but that's only to confirm the legacy path, which can easily be checked differently.

Building the WooCommerce plugin is required because it loads a copy of the plugin during unit testing. However that's not required to be a dev copy. So for unit testing it would just as happily run with the following two (as long as the WC version matched between the two):
- A WooCommerce plugin installed from WordPress.org
- A tests directory with those 4 required test files present.

I wouldn't mind changing bootstrap.php to check in a different location to encounter those test files (check in a different location and otherwise fallback to where it's looking now). That way we'd just need a mapping in wp-env to tell it where the test files are. You'd lose a little flexibility of being able to run unit tests with any WC branch, but would simplify the process.